src/options.js

```diff
@@ -27,8 +27,8 @@ const _defaultOptions = {
   stripPrompts: false,
   requestKernel: false,
   predefinedOutput: true,
-  mountStatusWidget: true,
-  mountActivateWidget: true,
+  mountStatusWidget: false,
```

Comment:
any reason not to make these true by default? If a page doesn't have the `thebe-activate` etc class, then this would just not have any effect right?

Reply:
yes, it's a no-op when they are not on the page. :+1: I'll set them to true